lib/pact/utils/metrics.rb

```diff
@@ -0,0 +1,62 @@
+require 'securerandom'
+require 'digest'
+require 'socket'
+require 'pact/version'
+
+module Pact
+  module Utils
+    class Metrics
+
+      def self.report_metric(event, category, action, value = 1)
+
+        if track_events?
+          Pact.configuration.output_stream.puts "WARN: Please note: we are tracking events anonymously to gather important usage statistics like Pact-Ruby version
+            and operating system. To disable tracking, set the 'pact_do_not_track' environment
```

Comment:
Upper case `PACT_DO_NOT_TRACK`.

Reply:
The other implementations have followed the original Pact.js implementation, which uses a lower case environment variable name

Reply:
Oh, I think you've misread the code. In Pact JS, you can set the `pact_do_not_track` _config_ item (i.e. npm config set pact_do_not_track true or in `package.json`). 

We still support the env var `PACT_DO_NOT_TRACK` also, see https://github.com/pact-foundation/pact-js-core/blob/pact-node/standalone/install.ts#L85.

Reply:
Ah, in that case we need to change the other implementations. Probably best to make it case-insensitive

Reply:
So, for clarity, Upper case `PACT_DO_NOT_TRACK` is what we want for consistency (including the warning). We can update rust later.